doc/chapters/design/index.rst

```diff
@@ -0,0 +1,39 @@
+SoftwareContainer Design
+************************
+
+SoftwareContainer has been designed as a building block enabling sandboxing of application contents in a larger system.
+
+The SoftwareContainer container achieves this by providing a structured way to limit what IPC mechanims are made available to each application. By targetting the IPC mechanisms rather than the applications or services directly, the sandboxing mechanism can be deployed in a non-intrusive way, i.e. completely transparent to both applications and services.
+
+Each IPC is managed by a *gateway*. The IPC needs of each service is described in *service manifest* files, defining *capabilities* and mapping these to individual *gateway configurations*. This can be individual objects or interfaces on d-bus, ipfilter rules, access to device nodes or something completely different.
+
+Each application is then associated with a set of *capabilities* in an *application manifest*. These are used to configure the sandbox in which the application is then launched.
```

Comment:
Sure, the capabilities are not implemented yet (in this component), but that's soon there so I do think it's relevant. I'm not so sure about the application manifests though. Perhaps it should be made more clear that it's not part of this components domain?